scripts/report_errors.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+export ERROR_LOG="/tmp/error.log"
+
+function report_error_to_github {
+    if [ $? -eq 0 ]; then
+        exit 0
+    fi
+
+    if [ ! -z $CI ]; then
```

Comment:
```suggestion
    if [ -n $CI ]; then
```

Reply:
That's a neat shortcut. Will get that updated.

Reply:
Just realized that change doesn't actually work as intended.